model/gen-ai/registry.yaml

```diff
@@ -13,9 +13,9 @@ groups:
               stability: experimental
               value: "openai"
               brief: 'OpenAI'
-            - id: vertex_ai
+            - id: vertex.ai
               stability: experimental
-              value: "vertex_ai"
+              value: "vertex.ai"
```

Comment:
I can't find any evidence of just "Vertex" without "AI" in GCP's docs, so `vertex_ai` actually seems appropriate to me. I can't imagine any other `vertex.*` values.

If that reasoning holds water to keep it as `vertex_ai`, I can verify with the team.

Reply:
I also believe we should not change this to be a namespace. Vertex AI seems to be the product name.